sttz.InstallUnity/sttz.InstallUnity.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net472</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
```

Comment:
Not sure why we need this, as I think it breaks Mac compability :p

But does not matter to us afaik

Reply:
We should keep Mac compatibility, so I will leave this open